client/src/components/Connections/components/Subgraph.js

```diff
@@ -186,9 +140,9 @@ export default compose(
       }
       if (checkShaking(pointer.canvas)) {
         const subgraphId =
-          props.entity2.query.length > 0
-            ? `${props.entity1.eids.join()}-${props.entity2.eids.join()}`
-            : `${props.entity1.eids.join()}`
+          props.notable
```

Comment:
So notable exist only to make these more readable?
We could compute `notable` from `eids1` and `eids2`(or lack there of) in i.e. `withProps`.
Or better yet calculate `subgraphId`(as well/instead of) as that appears to be the main point and is done in 2 places. That way we have one less prop to define and more readable code.
What's your take?

Reply:
Subgraph doesn't take `entity1` and `entity2` for props anymore, so it can't compute the value. And we can't check `eids2.length` instead, because maybe none were found. That's why we need to pass `notable`. Also, it makes it more clear